handler/setup.go

```diff
@@ -200,6 +202,34 @@ func getDockerSockVolume() *spec.Volume {
 	}
 }
 
+func appendConfigVolume(config *config.Config) []*spec.Volume {
```

Comment:
Rename the function to `getConfigVolumes`

Reply:
oh yes, pushed it